pkg/kubelet/apis/cri/v1alpha1/runtime/constants.go

```diff
@@ -25,3 +25,23 @@ const (
 	// NetworkReady means the runtime network is up and ready to accept containers which require network.
 	NetworkReady = "NetworkReady"
 )
+
+// StreamType is the type of the stream in CRI container log.
+type StreamType string
```

Comment:
StreamType seems too general, rename it to LogStreamType?

Reply:
Will do.

Reply:
Done.